### DISCUSSION TITLE
这个广告叫什么名字啊

### DISCUSSION THREAD

**Discussion by sqq135** [the1812/Bilibili-Evolved#4758] [提问 / Q&A]
![image](https://github.com/the1812/Bilibili-Evolved/assets/78015121/971d5a44-2bc5-46e5-8d5a-7c208d66e9c1)
如题，想要找个相关的插件，但是不知道目标广告的分类。如果指点用哪个插件就更好了

**Comment by the1812** [comment]
这个应该不算广告

**Reply by sqq135** [reply]
我记得之前是屏蔽的，但是忘了什么时候改了啥配置就不屏蔽了，关于这个的关键词是什么啊，我去搜搜

**Reply by the1812** [reply]
我这相关的只有个 `删除广告` 的功能

**Reply by Tinhone** [reply]
可以把屏蔽功能加到简化首页里？

**Reply by Geeyun-JY3** [reply]
我记得`简化首页`就是应该能隐藏原版首页的各种元素的吧，只是该组件的大部分选项在现在这个无限瀑布流首页基本都失效了，目前应该就只剩`分区栏`还没失效了。
而且我没记错的话中`活动/热门视频`这个选项当初就是能直接隐藏掉轮播横幅和旁边的推荐视频（曾是全站热门视频）这一整行的。
另外印象中现在好像是有不少脚本或扩展就能单独隐藏这个首页轮播横幅的，要做的话就不要还和以前一样之间干掉一整行了。
![image](https://github.com/the1812/Bilibili-Evolved/assets/49396421/a398a81e-b1b0-4dda-8d43-6ce8780200a5)

**Comment by AquaMT** [comment]
我自己在用的浏览器插件adguard，可以选择元素然后屏蔽
![QQ截图20240525233311](https://github.com/the1812/Bilibili-Evolved/assets/55498866/d7a1aafa-f470-4093-aa6b-9f7f1af0f03f)
